sonar-db/src/main/java/org/sonar/db/permission/UserPermissionMapper.java

```diff
@@ -27,22 +27,15 @@
 
 public interface UserPermissionMapper {
 
-  /**
-   * List of user permissions ordered by alphabetical order of user names
-   *
-   * @param query non-null query including optional filters.
-   * @param userLogins if null, then filter on all active users. If not null, then filter on logins, including disabled users.
-   *                   Must not be empty. If not null then maximum size is {@link org.sonar.db.DatabaseUtils#PARTITION_SIZE_FOR_ORACLE}.
-   */
-  List<ExtendedUserPermissionDto> selectByQuery(@Param("query") PermissionQuery query, @Nullable @Param("userLogins") Collection<String> userLogins, RowBounds rowBounds);
+  List<UserPermissionDto> selectByQuery(@Param("organizationUuid") String organizationUuid, @Param("query") PermissionQuery query, @Nullable @Param("userLogins") Collection<String> userLogins, RowBounds rowBounds);
```

Comment:
there is. I prefer moving Javadoc of mapper to DAO so that it's available to callers.